src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -886,6 +923,80 @@ public void saveResolutions(int series)
 
   }
 
+  private void saveHCSMetadata(N5Writer n5, IMetadata meta) throws IOException {
+    if (noHCS) {
+      return;
+    }
+
+    // assumes only one plate defined
+    int plate = 0;
+    Map<String, Object> plateMap = new HashMap<String, Object>();
+
+    plateMap.put("name", meta.getPlateName(plate));
+
+    List<Map<String, Object>> columns = new ArrayList<Map<String, Object>>();
+    for (int c=0; c<meta.getPlateColumns(plate).getValue(); c++) {
```

Comment:
After 30min of conversion of a plate, the call ended with a `java.lang.NullPointerException` while trying to write the HCS metadata. Now I understand better why https://github.com/ome/bioformats/pull/3638 is required :)

Two  questions:
- does this mean progress is blocked on having https://github.com/ome/bioformats/pull/3638 available in an mainline release?
- or do we need to implement some fallback anyways for the readers where this metadata  might be lacking?